.circleci/config.yml

```
@@ -724,6 +725,136 @@ jobs:
+      - run:
+          name: Start Appium server
+          command: appium --base-path /wd/hub
+          background: true
+      - run:
+          name: Start Metro
+          command: |
+            cd packages/rn-tester
+            yarn start
+          background: true
+      - brew_install:
+          package: cmake
+      - setup_ruby:
+          ruby_version: << parameters.ruby_version >>
+      - run: 
+          name: Install Bundler
+          command: |
+            cd packages/rn-tester
+            bundle check || bundle install
+            bundle exec pod setup
+            bundle exec pod install --verbose
+      - run:
+          name: Boot iOS Simulator
+          command: source scripts/.tests.env && xcrun simctl boot "$IOS_DEVICE" || true
+      - run:
+          name: Build app
+          command: |
+              xcodebuild build \
+                -workspace packages/rn-tester/RNTesterPods.xcworkspace \
+                -configuration Debug \
+                -scheme RNTester \
+                -sdk iphonesimulator \
+                -derivedDataPath /tmp/e2e/
+      - run:
+          name: Move app to correct directory
+          command: mv /tmp/e2e/Build/Products/Debug-iphonesimulator/RNTester.app packages/rn-tester-e2e/apps/rn-tester.app
+      - run:
+          name: Check Appium server status
+          command: |
+            if ! nc -z 127.0.0.1 4723; then
+              echo Could not find Appium server!
+              exit 1
+            fi
+      - run:
+          name: Run E2E tests
+          command: | 
+            cd packages/rn-tester-e2e 
+            yarn test-ios-e2e
+
+  # -------------------------
+  #     JOBS: Android E2E Tests
+  # -------------------------
+  test_e2e_android:
+    executor:
+        name: android/android-machine
+        tag: 2023.07.1
+    steps:
+      - checkout_code_with_cache
+      - run_yarn   
+      - android/create-avd:
+          avd-name: e2e_emulator
+          system-image: system-images;android-33;google_apis;x86_64
+          install: true
+      - android/start-emulator:
+          avd-name: e2e_emulator
+          no-window: true
+          restore-gradle-cache-prefix: v1a
+          post-emulator-launch-assemble-command: ""  
+      - run:
+          name: Install appium
+          command: npm install appium@2.0.0-rc.4 -g
+      - run:
+          name: Install appium drivers
+          command: |
+            appium driver install uiautomator2
+            appium driver install xcuitest
+      - run:
+          name: Start Appium server
+          command: appium --base-path /wd/hub
+          background: true
+      - run:
+          name: Start Metro
+          command: |
+            cd packages/rn-tester
+            yarn start
+          background: true
+      - attach_workspace:
+          at: . 
+      - run:
+          name: Build app
+          command: |
+            ./gradlew :packages:rn-tester:android:app:assembleHermesDebug                   
```

```suggestion
            ./gradlew :packages:rn-tester:android:app:assembleHermesDebug -PreactNativeArchitectures=x86_64
```

or `x86` based on which Arch we pick